src/config/config.ts

```diff
@@ -43,9 +43,32 @@ export const getPullZoneConfig = async () => {
     validator: validateDigitString,
     errorLogMessage: "The pull-zone-id should only contain digits.",
   });
+
+  const requestTimeout = await getInputWrapper({
+    inputName: "request-timeout",
+    inputOptions: { required: false },
+    validator: validatePositiveInteger,
+    transformInput: async (input: string) => {
+      const value = Number(input);
```

Comment:
Same reasoning as the previous comment about the retryLimit.

```suggestion
      transformInput: async (input: string) => Number(input),
```